src/Integration.Vsix.UnitTests/Integration.Vsix.UnitTests.csproj

```diff
@@ -28,6 +28,43 @@
     </Content>
   </ItemGroup>
   
+  <ItemGroup>
+    <None Remove="CFamily\CLangAnalyzerRequest.json" />
+    <None Remove="CFamily\IntegrationTests\CLangAnalyzerTestFile_NoIssues_EmptyFile.txt" />
+    <None Remove="CFamily\IntegrationTests\CLangAnalyzerTestFile_NoIssues_EmptyFile_response.json" />
+    <None Remove="CFamily\IntegrationTests\CLangAnalyzerTestFile_OneIssue.txt" />
+    <None Remove="CFamily\IntegrationTests\CLangAnalyzerTestFile_OneIssue_response.json" />
+    <None Remove="CFamily\IntegrationTests\CLangAnalyzerTestFile_TwoIssues.txt" />
+    <None Remove="CFamily\IntegrationTests\CLangAnalyzerTestFile_TwoIssues_response.json" />
+  </ItemGroup>
+  
+  <ItemGroup>
+    <Content Include="CFamily\IntegrationTests\CLangAnalyzerTestFile_NoIssues_EmptyFile_response.json">
```

Comment:
Do you need to change the ItemGroup type from `None` to `Content`, or do you just need to add the metadata property `CopyToOutputDirectory`?
If you just need to update the metadata you can use `Update` instead `Remove` + `Include` in SDK-style projects: https://docs.microsoft.com/en-us/visualstudio/msbuild/item-element-msbuild?view=vs-2019#attributes-and-elements